cli/file_fetcher.rs

```diff
@@ -583,7 +601,9 @@ impl FileFetcher {
       specifier
     );
     let scheme = get_validated_scheme(specifier)?;
-    options.permissions.check_specifier(specifier)?;
+    if let Some(permissions) = &options.permissions {
+      permissions.check_specifier(specifier)?;
+    }
```

Comment:
Again for https://github.com/denoland/deno/pull/25469 this will have to change again; maybe don't use `Option<>` here and always do the check?